pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
   sdk: '>=2.7.0 <3.0.0'
```

Comment:
Any particular reason not to bump the min SDK? Is it important that `dart_style` gets version updates on older SDKs?

Reply:
Not really. Went ahead and bumped it (somewhat arbitrarily) to 2.9.0.